lib/Cargo.toml

```diff
@@ -20,6 +20,7 @@ default = []
 rocksdb-datastore = ["rocksdb", "bincode"]
 test-suite = []
 bench-suite = []
+rocksdb-multi-threaded = ["rocksdb/multi-threaded-cf"]
```

Comment:
No need to add this, since multi-threaded CFs are just going to make things more complicated for us without any upside.

Reply:
Agree, but this PR name need some modification